src/handler.ts

```diff
@@ -0,0 +1,59 @@
+import {Response} from "express";
+import axios, {AxiosResponse} from "axios";
+import {NODES} from "./config";
+
+type Url = string; 
+type Blockchain = string;
+type Net = string;
+
+// Define constants
+const MAX_RETRIES: number = 5;
+const RESPONSE_SUCCESS_STATUS: number = 200; 
+
+// Handles proxy requests
+export const handler = async (req: any, res: Response) => {
+  req.current = req.current ? req.current : 0;
+
+  // Destructure props from request object
+  const { method, url, headers, body } = req;
+
+  /* 
+  Read parameters
+  Blockchain is required, net defaults to mainnet if not provided
+  */
+
+  const blockchain: Blockchain = req.params.blockchain.toLowerCase(); 
+  const net: Net = req.query.net ? req.query.net.toLowerCase() : "mainnet"; // Default to mainnet
+
+  const blockchainNodes: Array<string> = NODES[blockchain][net];
+
+  // Select server
+  const server: Url = blockchainNodes[req.current] !== undefined ? blockchainNodes[req.current] : blockchainNodes[0];
+
+	try {
+    // Do request
+    const response: AxiosResponse = await axios({
+      url: `${server}${url.replace(`/${req.params.blockchain}`, '')}`,
+      method: method,
+      // headers: headers, /* TODO: ERR_TLS_CERT_ALTNAME_INVALID */
+      data: body,
+    });
+
+    // Send response data back to original caller
+    if (response.status === RESPONSE_SUCCESS_STATUS) {
+      res.send(response.data);
+    }
```

Comment:
What if response.status !== success ? The connection would be hanging on until timeout from the client. We should send something, or remove this condition and just return something like:

`res.status(response. status).send(response.data) `

plus headers etc.